### DISCUSSION TITLE
Для провайдера добавить признак FMC

### DISCUSSION THREAD

**Discussion by jorikfon** [mikopbx/Core#141] [Ideas (Идеи)]
Если от провайдера приходят длинные и короткие номера, то прорабатывать их как входящие или как исходяшие соответвенно.
Насколько это можеть быть полезно в базовой поставке? 
Для объединения 2-х mikopbx может быть полезно?

**Comment by boffart** [comment]
Ты имеешь в виду did? 
Если опцию добавить, то может быть полезно, не описывать все маршруты. 
Хотя может и навредить, если логин учетки короткий.
